WISE-Community/WISE
# Implemented the 'Fill X Number of Rows' in a Table constraint.

Here's how to test
1. Create a Table component.
2. Add a "Fill X Number of Rows" constraint on that step and make it point to the Table component.
3. Make sure the "Required Number of Filled Rows" works properly.
4. Make sure the "Table Has Header Row" works properly. For example, if the table has a header row and the constraint requires 5 filled rows, the table will actually require a total of 6 filled rows.
5. Make sure the "Require All Cells In a Row To Be Filled" works properly. If this is set to false, it means a row is considered "Filled" if at least one cell has a value.

Closes #1731
